packages/swap/src/event-handlers/v140/boostPoolCreated.ts

```diff
@@ -0,0 +1,39 @@
+import { z } from 'zod';
+import { assetAndChain, number } from '@/shared/parsers';
+import { screamingSnakeToPascalCase } from '@/shared/strings';
+import { EventHandlerArgs } from '..';
+import { InternalAsset } from '../../client';
+
+const boostPoolCreatedEventSchema = z.object({
+  boostPool: z.object({
+    asset: assetAndChain.transform((chainAsset) => ({
+      ...chainAsset,
+      asset: screamingSnakeToPascalCase(chainAsset.asset),
+    })),
+    tier: number,
+  }),
+});
+
+export const boostPoolCreated = async ({ prisma, event }: EventHandlerArgs) => {
+  const {
+    boostPool: { asset: chainAsset, tier },
+  } = boostPoolCreatedEventSchema.parse(event.args);
+
+  await prisma.boostPool.upsert({
```

Comment:
should we expect to receive this event multiple times for the same asset? maybe it's better to throw error in that case 🤔

Reply:
Who knows 😅 
I think its better to ignore it rather than the server crash looping